src/core/frameworks/redux/index.ts

```diff
@@ -12,10 +17,11 @@ export const getStore = (extraArgument?: any) => {
     theme: cloneDeep(themeInitialState),
     wallet: cloneDeep(walletInitialState),
     settings: cloneDeep(settingsInitialState),
+    notifications: cloneDeep(notificationsInitialState),
   };
   const persistConfig = {
     namespace: 'yearn',
-    states: ['theme', 'wallet.name', 'settings', 'network'],
+    states: ['theme', 'wallet.name', 'settings', 'network', 'notifications.dismissedMessageIds'],
```

Comment:
this is going to make redux-localstorage-simple only load dismissedMessageIds into the store and not the other attributes from that reducer, you need to handle this by adding a preloadedState.

Take a look at https://github.com/kilkelly/redux-localstorage-simple#advanced-usage

Reply:
@xgambitox in the same file in like 56 there's a property inside configureStore `preloadedState: merge(initialState, persistedState)` does this handle the preloaded state?

Reply:
@TacosTonight correct, that should do.